app/src/main/java/de/westnordost/streetcomplete/quests/QuestsModule.kt

```diff
@@ -491,6 +492,7 @@ fun questTypeRegistry(
     150 to AddBuildingType(),
     151 to AddBuildingLevels(),
     152 to AddRoofShape(countryInfos, countryBoundariesFuture),
+    200 to AddRoofColour(countryInfos, countryBoundariesFuture),
```

Comment:
Index/sort order is currently a placeholder. I'm not sure where this should be placed in practice. 🤷